neurodamus/ngv.py

```diff
@@ -255,95 +230,40 @@ def finalize(self, astrocyte, base_Seed, *, base_connections=None, **kw):
         """Bind each glia connection to synapses in connections target cells via
         the assigned unique gid.
         """
-        # TODO: Currently it receives the base_connections object to look (bin search)
-        # for the sinapse to attach to. However since target cells and Glia might be
-        # distributed differently across MPI ranks, this is bound to work in a single rank.
-        # For the moment we fallback to using the original synapse id.
-
         self._netcons = []
-        sections_glut = astrocyte.sections_glut
-        n_bindings = 0
         pc = Nd.pc
 
-        def ustate_event_handler2(syn_gid):
-            return lambda: print("GOOD netcon event 2. Spiking via v-gid: " + str(syn_gid))
-
         if GlobalConfig.debug_conn:
             if GlobalConfig.debug_conn == [self.tgid]:
                 logging.debug("Finalizing conn %s. N params: %d", self, len(self._synapse_params))
             elif GlobalConfig.debug_conn == [self.sgid, self.tgid]:
                 logging.debug("Finalizing conn %s. Params:\n%s", self, self._synapse_params)
 
         for syn_params in self._synapse_params:
-            if USE_COMPAT_SYNAPSE_ID:
-                syn_gid = 1_000_000 + syn_params.synapse_id
-            else:
-                tgid_conns = base_connections.get(syn_params.connected_neurons_post)
-                syn_gid = self._find_neuron_endpoint_id(syn_params, tgid_conns)
-                if syn_gid is None:
-                    continue
+            syn_gid = self.syn_gid_offset + syn_params.synapse_id
 
+            # netcon to GlutReceive
             glut_idx = int(syn_params.astrocyte_section_id)
-            glut_obj = sections_glut[glut_idx]
+            glut_obj = astrocyte.get_glut(glut_idx)
             netcon = pc.gid_connect(syn_gid, glut_obj)
-            netcon.delay = 0.05
-
-            netcon.record(ustate_event_handler2(syn_gid))
-
+            netcon.delay = self.netcon_delay
+            netcon.record(
+                lambda syn_gid=syn_gid: print(f"GOOD netcon event 2. Spiking via v-gid: {syn_gid}")
```

Comment:
These printed messages seem like they were for debugging. Can @dkeller9 confirm if they should be removed in production?

Reply:
I see below a similar construct is used, but checks for
`if GlobalConfig.verbosity >= LogLevel.DEBUG:`
Maybe that should also be employed here?